### PR TITLE
Add 2 New Haven, CT Hackathons

### PR DESCRIPTION
**Pull request by hydrosquall**
Added Healthcare Hackathon (1.22-1.24) and Hack the Brain in New Haven, CT for next semester

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +52,11 @@ Inspired by the [Developer Conferences](https://github.com/MurtzaM/Developer-Con
 | [PennApps XIII](http://2016s.pennapps.com/) | Philadelphia, PA | 1.22 - 1.24 |
 | [UofTHacks](https://uofthacks.com/) | Toronto, Canada | 1.22 - 1.24 |
 | [SwampHacks](http://swamphacks.com/) | Gainesville, FL | 1.23 - 1.24 |
+| [Yale Healthcare Hackathon](http://www.yalehackhealth.org/) | New Haven, CT | 1.22-1.24 |
```
Comment (japacible) [inline README.md]: Please move this above SwampHacks.

super nit: please add spacing for the date: "1.22 - 1.24". Thanks!

**Comment by hydrosquall** [comment]
@japacible ok, fixed!

File: README.md [hunk 2]
```diff
@@ -51,10 +51,12 @@ Inspired by the [Developer Conferences](https://github.com/MurtzaM/Developer-Con
 | [IrishHacks](http://irishhacks.org/) | South Bend, IN | 1.22 - 1.24 |
 | [PennApps XIII](http://2016s.pennapps.com/) | Philadelphia, PA | 1.22 - 1.24 |
 | [UofTHacks](https://uofthacks.com/) | Toronto, Canada | 1.22 - 1.24 |
+| [Yale Healthcare Hackathon](http://www.yalehackhealth.org/) | New Haven, CT | 1.22 - 1.24 |
```
Comment (hydrosquall) [inline README.md]: Changed order of Healthcare Hack and added date spacing

**Comment by japacible** [comment]
Thanks!

**Pull request merged**